addons/website_crm_partner_assign/static/src/js/crm_partner_assign.js

```diff
@@ -252,9 +254,11 @@ publicWidget.registry.crmPartnerAssign = publicWidget.Widget.extend({
             $('.edit_opp_form .activity_summary').val($selected.attr('activity_summary'));
         }
         if ($selected.attr('delay_count')) {
-            var now = moment();
-            var date = now.add(parseInt($selected.attr('delay_count')), $selected.attr('delay_unit'));
-            $('.edit_opp_form .activity_date_deadline').val(date.format(time.getLangDateFormat()));
+            var now = DateTime.now();
+            const value = +$selected.attr('delay_count');
+            const unit = $selected.attr('delay_unit');
+            var date = now.plus({[unit] : value});
+            $('.edit_opp_form .activity_date_deadline').val( serializeDate(date) );
```

Comment:
probably false: serializeDate does not take into account the user's localization settings !
use formatDate